src/mantarray_desktop_app/process_monitor.py

```diff
@@ -444,14 +440,14 @@ def _commands_for_each_run_iteration(self) -> None:
             _get_dur_since_last_barcode_clear(self._last_barcode_clear_time)
             >= BARCODE_POLL_PERIOD
         ):
-            to_ok_comm = (
-                process_manager.queue_container().get_communication_to_ok_comm_queue(0)
+            to_instrument_comm = process_manager.queue_container().get_communication_to_instrument_comm_queue(
+                0
             )
             barcode_poll_comm = {
                 "communication_type": "barcode_comm",
                 "command": "start_scan",
             }
-            to_ok_comm.put(barcode_poll_comm)
+            to_instrument_comm.put(barcode_poll_comm)
```

Comment:
As part of switching the get to getnowait we should consider what to do with these puts that currently have no timeout